### PR TITLE
Closes #3156: Add site link to rack reservations overview

### LINKED ISSUE #3156: Extend Rack Reservations Table-View with Site Information
### Environment
* Python version: 3.6  
* NetBox version: 2.5.11 

### Proposed Functionality
On  the ``Rack Reservations`` Overview-Page ``/dcim/rack-reservations/`` we should Display the ``Site`` Name in Table-Overview too.

### Use Case
If you check your ``Rack Reservations`` you could see the ``Rack`` and maybe the ``Tenant`` if Information is given.
To make the overview even easier, the first column should contain the corresponding ``Site``. 

image

### Database Changes
None.

### External Dependencies
None.

### PR DESCRIPTION
**Pull request by MarcHagen**

### CODE REVIEW

File: netbox/dcim/views.py [hunk 1]
```diff
@@ -458,7 +458,7 @@ class RackBulkDeleteView(PermissionRequiredMixin, BulkDeleteView):
 #
 
 class RackReservationListView(ObjectListView):
-    queryset = RackReservation.objects.all()
+    queryset = RackReservation.objects.all().select_related('rack__site')
```
Comment (jeremystretch) [inline netbox/dcim/views.py]: You can remove `.all()` when it's followed by another method; it's effectively a no-op.

**Review by jeremystretch — CHANGES_REQUESTED** [review]
Looks good, thanks! Just spotted a couple minor tweaks.

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,7 @@
 * [#2813](https://github.com/digitalocean/netbox/issues/2813) - Add tenant group filters
 * [#3085](https://github.com/digitalocean/netbox/issues/3085) - Catch all exceptions during export template rendering
 * [#3138](https://github.com/digitalocean/netbox/issues/3138) - Add 2.5GE and 5GE interface form factors
+* [#3152](https://github.com/digitalocean/netbox/issues/3152) - Add site link to rack reservations overview
```
[the remark this answers is not in the document]
Comment (TakeMeNL) [reply CHANGELOG.md]: Ah yes, at first i had the wrong issue number. pretty embarrassed about that.
Fixed in 1ff7e11

File: netbox/dcim/views.py (hunk 1 above)
Comment (TakeMeNL) [reply netbox/dcim/views.py]: Fixed in 1ff7e11

**Comment by TakeMeNL** [comment]
Thank you for the review! I'm still learning python and Django so try to do smaller issues.
It's fun to figure out how stuff works 🎉

**Comment by jeremystretch** [comment]
Ignore the test failures. This has been a recurring issue with our CI.

**Pull request merged**